score/tile/generate.py

```diff
@@ -0,0 +1,91 @@
+import os
+from pathlib import Path
+import shutil
+
+from etl.sources.census.etl_utils import get_state_fips_codes
+
+
+def generate_tiles(data_path: Path):
+
+    # remove existing mbtiles file
+    mb_tiles_path = data_path / "tiles" / "block2010.mbtiles"
```

Comment:
Nit: with all the ETL functions, we separated out the cleanup script from the generation script. Is it worth doing that here for consistency? We could always create a single function afterwards that calls both (e.g., `cleanup_and_generate_tiles`.

Reply:
For the generate tiles, in my opinion, we do want to tie the clear up and generation on the same script, as the moment you're invoking this command you want to nuke whatever is there totally, versus checking if some tiles exist and others not.